packages/javascript-sdk/src/fr-webauthn/index.ts

```diff
@@ -273,19 +290,33 @@ abstract class FRWebAuthn {
     try {
       const clientDataJSON = arrayBufferToString(credential.response.clientDataJSON);
       const assertionResponse = credential.response as AuthenticatorAssertionResponse;
-      const authenticatorData = new Int8Array(assertionResponse.authenticatorData).toString();
+      const authenticatorData = new Int8Array(
+        assertionResponse.authenticatorData,
+      ).toString() as AttestationType;
       const signature = new Int8Array(assertionResponse.signature).toString();
 
       // Current native typing for PublicKeyCredential does not include `userHandle`
       // eslint-disable-next-line
       // @ts-ignore
       const userHandle = arrayBufferToString(credential.response.userHandle);
 
-      let stringOutput = `${clientDataJSON}::${authenticatorData}::${signature}::${credential.id}`;
+      let stringOutput =
+        `${clientDataJSON}::${authenticatorData}::${signature}::${credential.id}` as DeviceName<
```

Comment:
Had to force this type, theres probably a way to create a string builder that outputs as the type though that may fit better.